packages/core/legos/src/sources.ts

```diff
@@ -10,6 +10,7 @@ import * as bitex from '@chainlink/bitex-adapter'
 import * as bitso from '@chainlink/bitso-adapter'
 import * as blockchain_com from '@chainlink/blockchain.com-adapter'
 import * as blockchair from '@chainlink/blockchair-adapter'
+import * as blockstream from '@chainlink/blockstream-adapter'
```

Comment:
Something I noticed is that the blockstream test cases would cause the CI to fail even though I didn't make any changes. Also noticed that it wasn't included in the original legos package on EAEE branch, should I remove it from here?

Reply:
fixed by splitting test cases into `integration` and `unit` folder